packages/core/src/components/axes/brush.ts

```diff
@@ -0,0 +1,136 @@
+// Internal Imports
+import { Component } from "../component";
+import { Tools } from "../../tools";
+import { ScaleTypes } from "../../interfaces";
+import { DOMUtils } from "../../services";
+
+// D3 Imports
+import { extent } from "d3-array";
+import { brushX } from "d3-brush";
+import { event } from "d3-selection";
+import { scaleTime } from "d3-scale";
+
+export class Brush extends Component {
```

Comment:
Change `Brush` to `ChartBrush`.
This class is used for handle brush events in chart.
It's only used by zoom in for now, but it could be improved for other brush events in the future.